newIDE/app/src/Profile/SubscriptionDialog.js

```diff
@@ -31,7 +30,9 @@ import Window from '../Utils/Window';
 import Text from '../UI/Text';
 import GDevelopThemeContext from '../UI/Theme/ThemeContext';
 import { ColumnStackLayout, LineStackLayout } from '../UI/Layout';
+import RedeemIcon from '@material-ui/icons/Redeem';
```

Comment:
I wonder if one of those icons could be used:
<img width="32" alt="image" src="https://user-images.githubusercontent.com/32449369/201619725-a808599e-73a1-4e98-b964-8b053728fdaa.png">
<img width="43" alt="image" src="https://user-images.githubusercontent.com/32449369/201619749-8884bc70-9fcd-4da4-9a06-21b92032efda.png">
<img width="33" alt="image" src="https://user-images.githubusercontent.com/32449369/201619741-3937438b-f66f-4e60-961f-1b54d1ad1315.png">
<img width="30" alt="image" src="https://user-images.githubusercontent.com/32449369/201619760-20521c19-7e78-4fdc-8e67-0f845dbca184.png">

Reply:
Done, used the second one.